web/modules/custom/solution/solution.behat.inc

```diff
@@ -352,8 +308,53 @@ class SolutionSubContext extends DrupalSubContextBase implements DrupalSubContex
       'webdav creation' => 'field_is_webdav_creation',
       'webdav url' => 'field_is_webdav_url',
       'wiki' => 'field_is_wiki',
-      'groups audience' => 'og_group_ref',
+      'collection' => 'og_group_ref',
+    ];
+  }
+
+  /**
+   * Converts values from user friendly to normal machine values.
+   *
+   * @param array $fields
+   *    An array of fields keyed by field name.
+   *
+   * @return mixed
+   *    The array with the values converted.
+   *
+   * @throws \Exception
+   *    Throws an exception when a mapped value is not found.
+   */
+  private function convertValueAliases($fields) {
```

Comment:
Why is this private? Let's make it protected, there is usually no good reason to make anything private.